src/Controls/src/Core/Compatibility/Android/Renderers/MotionEventHelper.cs

```diff
@@ -23,7 +23,8 @@ public bool HandleMotionEvent(IViewParent parent, MotionEvent motionEvent)
 				return false;
 			}
 
-			return true;
+           // We should consider the InputTransparent for the VisualElement and decides to pass touch to the event
+			return ShouldPassThroughElement();
```

Comment:
I want to do more tests locally but ideally...

This implementation should be the same as the one in the Compatibility layer https://github.com/dotnet/maui/blob/734c3d26065e3ca712db111cf324e466d0e6064c/src/Compatibility/Core/src/Android/Renderers/MotionEventHelper.cs#L19

In cases like the Frame, the missing part is the NotifyFakeHandling that impact in the behavior of the DispatchTouchEvent method from an y View.

https://github.com/dotnet/maui/blob/734c3d26065e3ca712db111cf324e466d0e6064c/src/Compatibility/Core/src/Android/AppCompat/Platform.cs#L805